frontend/src/lib/components/CommandBar/CommandBar.tsx

```diff
@@ -7,24 +7,43 @@ import { commandBarLogic } from './commandBarLogic'
 import { BarStatus } from './types'
 
 import './index.scss'
-import SearchBar from './SearchBar'
-import { LemonModal } from '@posthog/lemon-ui'
-import ActionBar from './ActionBar'
+import { SearchBar } from './SearchBar'
+import { ActionBar } from './ActionBar'
 
-function CommandBar(): JSX.Element | null {
+const CommandBarOverlay = ({ children }: { children?: React.ReactNode }): JSX.Element => (
+    <div
+        className="fixed top-0 left-0 w-full h-full flex flex-col items-center justify-center"
```

Comment:
Doesn't have to be solved here but on smaller viewports I think a little margin would make sense and also the bottom bar should have overflow-x:auto
<img width="522" alt="Screenshot 2023-11-16 at 07 28 48" src="https://github.com/PostHog/posthog/assets/2536520/fa4350ff-854d-4ecb-9da9-c9a2bb0cf92b">

Reply:
Good points! Added these improvements as well.